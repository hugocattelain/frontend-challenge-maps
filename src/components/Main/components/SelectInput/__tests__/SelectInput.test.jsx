//Libraries
import React from 'react';
import { mount } from 'enzyme';

// Components
import { SelectInput } from '..';

describe('SelectInput', () => {
  const props = {
    options: [
      { text: 'All', value: 'all' },
      { text: 'Pizza', value: 'pizza' },
      { text: 'Burger', value: 'burger' },
      { text: 'Sushi', value: 'sushi' },
    ],
    value: '',
    onChange: jest.fn(value => value),
  };

  it('should render correctly', () => {
    const wrapper = mount(<SelectInput {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
