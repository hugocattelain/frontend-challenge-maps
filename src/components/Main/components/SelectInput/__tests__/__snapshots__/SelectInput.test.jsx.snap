// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectInput should render correctly 1`] = `ReactWrapper {}`;
